### DISCUSSION TITLE
TPA - 1era Entrega - Creación de Repo

### DISCUSSION THREAD

**Discussion by matiastito** [dds-jv/foro#412] [Consultas generales]
Buenas, espero estén bien, los molesto porque quería saber si para el TPA hay alguna restricción con respeto al repo, si hay que crearlo bajo alguna organización; obviamente lo hacemos con el user de la facu pero no sé si hay algún requerimiento adicioanl.

Muchas gracias!

**Comment by flbulgarelli** [comment]
¡Hola! Estaremos asignando en breves repositorios privados en la organización dds-utn.

**Comment by matiastito** [comment]
¡Gracias!
